plugins/kubernetes-api/ts/kubernetesApiHelpers.ts

```diff
@@ -268,16 +268,6 @@ module KubernetesAPI {
     };
   }
 
-  /**
-   * Filter a collection by label
-   **/
-  export function filterByLabel(objects: Array<any>, labelSelector: LabelMap) {
```

Comment:
The `labelsToString` and `selectorMatches` methods below may also be removed.

Reply:
The `labelFilterTextSeparator` variable may also be removed.

Reply:
removed